src/Digdir.Domain.Dialogporten.Infrastructure/Digdir.Domain.Dialogporten.Infrastructure.csproj

```diff
@@ -24,6 +24,8 @@
         <PackageReference Include="ZiggyCreatures.FusionCache" Version="2.2.0"/>
         <PackageReference Include="ZiggyCreatures.FusionCache.Backplane.StackExchangeRedis" Version="2.2.0"/>
         <PackageReference Include="ZiggyCreatures.FusionCache.Serialization.NeueccMessagePack" Version="2.2.0"/>
+        <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.3.0"/>
```

Comment:
Ikke optimalt å måtte dra i disse dependenciene i Infrastruktur, men trenger dem for å kunne konstruere en httpcontext slik at man får sendt requests.